models/user.go

```diff
@@ -1445,14 +1572,19 @@ func SyncExternalUsers() {
 						}
 						usr.IsActive = true
 
-						err = UpdateUserCols(usr, "full_name", "email", "is_admin", "is_active")
+						err = UpdateUser(usr)
```

Comment:
why update all the columns?

Reply:
@lunny @lafriks - right, got it. this update was unnecessary. now removed.